msteams-platform/task-modules-and-cards/cards/cards-actions.md

```diff
@@ -8,6 +8,8 @@ ms.date: 11/07/2024
 
 # Card actions
 
+[!INCLUDE [adaptive-card-redirect](../../includes/adaptive-card-redirect.md)]
```

Comment:
The banner doesn't work on this page which is about more than just Adaptive Cards. We already have links to the new site need to be under the "Adaptive Card actions" title, so that might be enough. But we could consider adding the banner under that title as well, maybe a short version of it without the image.

Reply:
Looks great